blocks/default-output-block/output-types/default.jsx

```diff
@@ -84,11 +63,131 @@ const comscoreNoScript = (accountId) => {
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());gtag('config', '${gaID}');
+  `;
+  return (
+    <>
+      <link rel="preconnect" href="https://www.googletagmanager.com/" />
+      <script async data-integration="googleAnalytics" src={`https://www.googletagmanager.com/gtag/js?id=${gaID}`} />
+      <script data-integration="googleAnalytics" dangerouslySetInnerHTML={{ __html: gaScript }} />
+    </>
+  );
+};
+
+const googleTagManagerScript = (gtmID) => {
+  if (!gtmID) {
+    return null;
+  }
+  const gtmScript = `
+    (function(w,d,s,l,i){
+      w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});
+      var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','${gtmID}');
+  `;
+  return (
+    <>
+      <link rel="preconnect" href="https://www.googletagmanager.com/" />
+      <script data-integration="googleTagManager" dangerouslySetInnerHTML={{ __html: gtmScript }} />
+    </>
+  );
+};
+
+const googleTagManagerNoScript = (gtmID) => {
+  if (!gtmID) {
+    return null;
+  }
+  return (
+    <noscript>
+      <iframe
+        title="gtm"
+        src={`https://www.googletagmanager.com/ns.html?id=${gtmID}`}
+        height="0"
+        width="0"
+        style={{
+          display: 'none',
+          visibility: 'hidden',
+        }}
+      />
     </noscript>
   );
 };
 
+const nativoIntegrationScript = (nativoIntegration) => {
+  if (!nativoIntegration) {
+    return null;
+  }
+  return (
+    <script async data-integration="nativo-ad" src="https://s.ntv.io/serve/load.js" />
+  );
+};
+
+const querylyCode = (querylyId, querylyOrg, pageType) => {
+  if (!querylyId) {
+    return null;
+  }
+  const querylyInit = `
+    window.addEventListener('DOMContentLoaded', (event) => {
+      queryly.init("${querylyId}", document.querySelectorAll("#fusion-app"));
+    });
+  `;
+  return (
+    <>
+      <link rel="preconnect" href="https://www.queryly.com/" />
+      <script defer data-integration="queryly" src="https://www.queryly.com/js/queryly.v4.min.js" />
+      <script data-integration="queryly" dangerouslySetInnerHTML={{ __html: querylyInit }} />
+      { pageType === 'queryly-search'
+        ? <script defer data-integration="queryly" src={`https://www.queryly.com/js/${querylyOrg}-advanced-search.js`} />
+        : null}
+    </>
+  );
+};
+
+const fontUrlLink = (fontUrl) => {
+  // If fontURL is an array, then iterate over the array and build out the links
+  if (fontUrl && Array.isArray(fontUrl) && fontUrl.length > 0) {
+    const fontLinks = [...new Set(fontUrl)].map((url, index) => (
+      <>
+        <link rel="prefetch" as="font" key={url} data-testid={`font-loading-url-prefetch-${index}`} href={`${url}&display=swap`} crossOrigin="anonymous" />
```

Comment:
This prefetch is not pre fetching as `font` the URL is to a CSS file. 
Also, prefetch's primary purpose is to fetch resources that may be required in the next navigation event.

Reply:
Since it's potentially preconnecting, I'm gonna go ahead and remove this, but MDN stated that the as=font was for @font-face declarations and not stylesheets... so I went with it.